lib/node_modules/@stdlib/math/base/special/sqrtpi/lib/main.js

```diff
@@ -0,0 +1,59 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2022 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var sqrt = require( '@stdlib/math/base/special/sqrt' );
+var PI = require( '@stdlib/constants/float64/pi' );
+
+
+// MAIN //
+
+/**
+* Computes the positive square root of the product of pi and a provided positive number.
+*
+* @type {Function}
+* @param {number} x - input value
```

Comment:
```suggestion
* @param {number} x - input value
```

The type annotation is not necessary here, as the type can be inferred from the `function foo() {}` signature.